contributed_definitions/NXcoordinate_system.nxdl.xml

```diff
@@ -3,7 +3,7 @@
 <!--
 # NeXus - Neutron and X-ray Common Data Format
 #
-# Copyright (C) 2014-2024 NeXus International Advisory Committee (NIAC)
+# Copyright (C) 2023-2025 NeXus International Advisory Committee (NIAC)
```

Comment:
That one was drafted with NXcoordinate_system_set in a pair but might not have been committed first like its partner _set, so using 2023 is fine certainly more better fitting than 2014.